contracts/pool/src/storage.rs

```diff
@@ -49,6 +49,8 @@ pub struct Config {
     pub max_allowed_spread_bps: i64,
     /// The maximum allowed percentage (in bps) for referral fee
     pub max_referral_bps: i64,
+    /// * `tolerance` - The smallest difference in a deposit we care about.
+    pub tolerance: i64,
```

Comment:
Please put more extensive comment plus name this variable more explicitely, `tolerance` hardcoded in a code is fine because you can see the use case right after; If we make it into a parameter, we need to make sure it's clear enough to not confuse other users.

Reply:
I see, added better commenting and renamed the value to tolerance_bps